test/test_utils.py

```diff
@@ -869,12 +868,18 @@ def test_load_standalone(self):
                 exec_path,
                 os.path.join(build_dir, f"standalone_load_test{ext}")
             )
-
+            env = os.environ.copy()
+            if IS_WINDOWS:
+                cudaPath = "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v" + os.environ.get('CUDA_VERSION', "")
```

Comment:
Isn't there a better way how to get path to CUDA? E.g. upstream or 3rd party CMake module, or at least use environment variable?